tensorboard/webapp/util/ui_selectors_test.ts

```diff
@@ -63,44 +54,39 @@ describe('ui_selectors test', () => {
 
   describe('#getCurrentRouteRunSelection', () => {
     it('returns selection map of current experiments', () => {
-      const state = {
-        ...buildStateFromAppRoutingState(
+      const state = buildMockState({
+        app_routing:
```

Comment:
As I mentioned elsewhere, moving from using functions like `...buildStateFromAppRoutingState` to using bare constant values like `'app_routing'` could probably use some discussion with the rest of the FE team.

The changes to use buildMockState() in this file are fine but its still unrelated to the rest of the PR as far as I can tell. (Perhaps it was necessary previously when you had also updated getRuns() to rely on Hparams state).